apps/dialtone-documentation/docs/design/spacing/index.md

```diff
@@ -0,0 +1,124 @@
+<p class="d-body-base">Consistent and intentional spacing decisions across pages create a predictable layout, reducing cognitive load and allowing users to focus on the content rather than the layout.</p>
+    </dt-stack>
+  </div>
+  <dt-stack class="d-gc2" direction="column" gap="500">
+    <iframe style="border: 0px; border-radius: 8px" width="100%" height="320" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Fproto%2FQe6cz41vPBozP4PhgGqFin/Docs-Protos?page-id=0%3A1&type=design&node-id=5-16175&viewport=702%2C117%2C0.97&t=LxszUdOGsEU9l3I5-8&scaling=min-zoom&starting-point-node-id=5%3A16175&hotspot-hints=0&hide-ui=1"></iframe>
+    <p class="d-body-base d-fc-tertiary">
+    <strong>Proximity groups</strong> are formed by placing elements close together. <strong>Delimited groups,</strong> on the other hand, are defined by clear visual boundaries, such as borders.
+    </p>
+    </dt-stack>
+</div>
+
+### Guiding the eye
+
+<div class="d-d-grid d-gg16 d-g-cols3 md:d-g-cols1" >
+  <div>
+    <dt-stack direction="column" gap="400">
+      <div>
+        <p>Control the amount of space around an element to affect its visual importance. Use larger spaces for more prominent elements and smaller for those of lower importance. Keep in mind the position of the elements will benefit the order of reading.
+      </p>
+      </div>
+      <div>
+        <p class="d-body-base">The empty areas around elements provide visual breathing room, enhancing readability and contrast.</p>
+      </div>
+    </dt-stack>
+  </div>
+  <div class="d-gc2">
+    <iframe style="border: 0px; border-radius: 8px" width="100%" height="320" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Fproto%2FQe6cz41vPBozP4PhgGqFin/Docs-Protos?page-id=0%3A1&type=design&node-id=5-16225&viewport=702%2C117%2C0.97&t=LxszUdOGsEU9l3I5-8&scaling=min-zoom&starting-point-node-id=5%3A16225&hotspot-hints=0&hide-ui=1"></iframe>
+    </div>
+</div>
+
+## Tokens
+
+Here are some frequently used tokens. For a complete list, visit the [Spacing Tokens](/tokens/spacing) section.
+
+<token-table category="space" :tokenList="tokenList" />
+
+WIP: Tokens decriptions/usage in progress.
+
+**Note:** By Feb 2024, we aim to integrate **spacing** units into Figma Variables. This will simplify the process of setting these tokens on space-between and paddings within your elements in your Figma files, making it easier for engineers to obtain the correct variable for each case.
+
+## Key Takeaways
+
+- Use predefined tokens, they help maintain consistency and make it easier to update designs.
+- Keep sizes and spacing consistent: This ensures a polished look across all devices.
+- Maintain a balanced visual hierarchy by using the modular scale for font, sizes and spacing.
```

Comment:
Did we mention anything about this in the document?

Reply:
Updated in the latest commit. Thank you!